### DISCUSSION TITLE
引用文献问题

### DISCUSSION THREAD

**Discussion by Dyz17392** [note286/xduts#188] [Q&A] [answered]
你好，我的参考文献不显示，具体原因为：
<img width="536" alt="image" src="https://github.com/note286/xduts/assets/128827163/8d2df72b-4042-4f64-b719-1e559827034a">
引用在文章中显示为
<img width="302" alt="image" src="https://github.com/note286/xduts/assets/128827163/93fb7f22-46d8-4a91-a6da-f0b117b3ba72">
请问如何解决？

**Comment by note286** [comment, marked as answer]
图里有提示，请使用`biber`编译。
